lib/mayaUsd/ufe/UsdUndoCreateGroupCommand.cpp

```diff
@@ -76,7 +85,7 @@ void UsdUndoCreateGroupCommand::execute()
         _groupCompositeCmd->append(setKindCmd);
         setKindCmd->execute();
     }
```

Comment:
On the plugin side, we only need to create the groupItem and set its Kind.  

On the Maya side, the insertion (a.k.a move ) happens with support for different flag options.